text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
```

Comment:
Just read this [article on Zenhub](https://www.zenhub.com/blog/how-to-use-epics-and-milestones/) about best practices in Epics, and naming each of these bullet points as a short version of a "job to be done" is something they suggest b/c it makes it super easy to convert this into an Epic (and as it happens, is also a best UX practice since each line will translate most easily to wireframes since these phrases describe what users ought to either "see" or be able to "do" (CTA)). For example:
Individuals Pages
Users who visit this page should be able to:
- see contributions made my the individual (plugins, PRs) and follow links to view details about those contributes
- see what sites the individual has built on Gatsby and visit those sites / see details about them (probably link to the site showcase)
- hire the person (if available)
- see whether the person is open for Gatsby projects or not
- find a link to the person's portfolio / outside website

Reply:
I love Jobs to Be Done! I'll update this.